### PR TITLE
wallet: add create auction txs

### PR DESCRIPTION
**Pull request by rithvikvibhu**
https://github.com/handshake-org/hsd/pull/529 added a way to create auction transactions in advance.
This PR adds it to WalletClient.

### CODE REVIEW

File: lib/wallet.js [hunk 1]
```diff
@@ -484,6 +484,22 @@ class WalletClient extends Client {
     return this.post(`/wallet/${id}/create`, options);
   }
 
+  /**
+   * Create pre-signed bid and reveal txs,
+   * fill, and optionally sign and broadcast.
+   * @param {Object} options
+   * @param {String} options.name
+   * @param {Number} options.bid
+   * @param {Number} options.lockup
```
Comment (pinheadmz) [inline lib/wallet.js:493]: might need to include `passphrase` as well, I see that as an option in hsd wallet/http.js

**Review by pinheadmz — CHANGES_REQUESTED** [review]
Great, thank you -- how about adding a CLI command in `hsw-cli` as well? You can use `mktx`/`createTX()` as a template I think. Be sure you add a help message describing the method. Finally, open a PR https://github.com/handshake-org/handshake-org.github.io adding CLI, cURL and JS methods to the API docs?

<3 <3 <3

...and by the way, if you enjoyed writing this PR there are a ton of new API endpoints that we need to add to the client: 

https://github.com/handshake-org/hs-client/issues/34

...and docs 😬 

https://github.com/handshake-org/handshake-org.github.io/issues/123

File: lib/wallet.js (hunk 1 above)
Comment (rithvikvibhu) [reply lib/wallet.js:493]: Added passphrase

**Comment by rithvikvibhu** [comment]
Turns out I had missed adding to `Wallet` class (the second half copy in the same file :confused:). Fixed that and added to hsw-cli.
```
$ mkauctiontxs [name] [bid] [lockup] [broadcast]: Create bid and reveal transactions.
```

Will update the command name if needed.

And docs updated in https://github.com/handshake-org/handshake-org.github.io/pull/148

**Pull request merged**